libminifi/include/core/Core.h

```diff
@@ -95,6 +98,62 @@ static inline std::string getClassName() {
 #endif
 }
 
+constexpr std::string_view removeStructOrClassPrefix(std::string_view input) {
+  constexpr std::string_view STRUCT = "struct ";  // should be static constexpr, but that is only allowed inside a constexpr function with std >= c++23
+  constexpr std::string_view CLASS = "class ";
```

Comment:
Would `constinit const` work to circumvent the limitation?

Reply:
No, because if `STRUCT` and `CLASS` are only `constinit`, not `constexpr`, then the result of the function is not `constexpr`.

This would work:
```c++
namespace detail {
  inline constexpr std::string_view STRUCT = "struct ";
  inline constexpr std::string_view CLASS = "class ";
}

constexpr std::string_view removeStructOrClassPrefix(std::string_view input) {
  for (auto prefix : { detail::STRUCT, detail::CLASS }) {
    if (input.find(prefix) == 0) {
      return input.substr(prefix.size());
    }
  }
  return input;
}
```
Would that be better?

Reply:
I'm neutral, pick what you think is best.

What about this, using literals inline?

```cpp
constexpr std::string_view removeStructOrClassPrefix(std::string_view input) {
  using namespace std::literals;
  for (auto prefix : { "struct "sv, "class "sv }) {
    if (input.find(prefix) == 0) {
      return input.substr(prefix.size());
    }
  }
  return input;
}
```

Reply:
I like this; changed to this version in 441b998f4fa682fa71345e1be8fafe8073c0cd32.